bears/setup.py.in

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+import locale
+
+import setuptools.command.build_py
+from setuptools import find_packages, setup
+
+try:
+    locale.getlocale()
+except (ValueError, UnicodeError):
+    locale.setlocale(locale.LC_ALL, 'en_US.UTF-8')
+
+if __name__== "__main__":
+	setup(name='$NAME',
+		  version=$VERSION,
+		  description='The $NAME bear for coala (http://coala.rtfd.org/)',
```

Comment:
you mean the docstring? i have tohught about that, but for some bears that description is reaaallly loong or helpful for pypi